centreon/tests/php/mock/Object/CentreonTimeperiod.php

```
@@ -0,0 +1,75 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Centreon\Test\Mock\Object;
+
+class CentreonTimeperiod extends BaseObject
+{
+    public function getTimperiodIdByName($tp_name)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$tp_name'. (Unused Code Rules)

(UnusedFormalParameter)